cucumber.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 3.0'
   s.required_rubygems_version = '>= 3.2.8'
 
+  s.add_dependency 'base64', '~> 0'
```

Comment:
I'm not a fan of single number relations, because they don't work as advertised. Can we use an `x.y` reference for these

Reply:
Done